geode-docs/getting_started/intro_to_clients.html.md.erb

```diff
@@ -0,0 +1,353 @@
+with very rudimentary connect, put, get, and then a reference to more in-depth docs and examples on
+how to use the client.
+
+## Installing Apache Geode
+
+You can download Apache Geode from the website, run a Docker image, or install with homebrew on OSX.
+
+## Start an Apache Geode Cluster
+
+With a path that contains the bin directory of the installation, start a locator and server.
+
+```
+$ gfsh 
+gfsh> start locator
+gfsh> start server
+```
+
+Create a region
+
+```
+gfsh> create region --name=helloWorld --type=PARTITION 
+```
+
+Shutting Down Apache Geode
+
+```
+gfsh> shutdown --include-locators=true
+```
+
+## Apache Geode Java Client
+
+**Maven Dependencies**
+
+Declare the following dependency and replace $VERSION, with the version of Geode that you have installed.
+
+```
+<dependencies>
+     <dependency>
+        <groupId>org.apache.geode</groupId>
+        <artifactId>geode-core</artifactId>
+        <version>$VERSION</version>
+    </dependency>
+</dependencies>
+```
+
+**Gradle Dependencies**
+
+``
+dependencies {
+  compile "org.apache.geode:geode-core:$VERSION"
+}
+``
+
+Connect to Geode from your application
+ 
+```
+import org.apache.geode.cache.client.ClientCache;
+import org.apache.geode.cache.client.ClientCacheFactory;
+
+public class HelloWorld {
+
+ public static void main(String[] args) {
+  ClientCache cache = new ClientCacheFactory().addPoolLocator("127.0.0.1", 10334).create();
+
+   System.out.println(cache.getDefaultPool().getLocators());
+   
+cache.close();
+ }
+}
+```
+
+The information printed out should match the host and port of your Geode instance locators and should resemble
+
+```
+[/127.0.0.1:10334]
+```
+
+### Simple Put and Get with Geode Java client
+ 
+```
+public static void main(String[] args) {
+ClientCache cache = new ClientCacheFactory().addPoolLocator("127.0.0.1", 10334).create();
+
+ Region<String, String>
+     helloWorldRegion =
+     cache.<String, String>createClientRegionFactory(ClientRegionShortcut.PROXY).create("helloWorld");
+
+ helloWorldRegion.put("1", "HelloWorldValue");
+ String value1 = helloWorldRegion.get("1");
+ System.out.println(value1);
+
+ cache.close();
+}
+```
+
+Build and run the application.  This puts the key ‘1’ with a value of ‘HelloWorldValue’ into the
+‘helloWorld’ region, then retrieves and prints the value of key ‘1’.
+
+### Additional Resources
+
+- [Apache Geode Examples GitHub Repo](https://github.com/apache/geode-examples)
+- [Apache Geode Documentation](https://geode.apache.org/docs/)
+- [Apache Geode Javadocs](https://geode.apache.org/releases/latest/javadoc/index.html)
+
+## Spring Boot Client For Apache Geode
+
+Spring Boot for Apache Geode provides a powerful abstraction that simplifies the developer
+experience when using Spring Boot and Apache Geode.  The best way to get started with Spring Boot
+for Apache Geode, is by creating a project on start.spring.io.
```

Comment:
I would convert the text "start.spring.io" to a link to that page. And in line 186 too.

Reply:
done